src/payment/strategies/klarna/klarna-script-loader.ts

```diff
@@ -1,17 +1,25 @@
 import { ScriptLoader } from '@bigcommerce/script-loader';
 
 import KlarnaCredit from './klarna-credit';
+import KlarnaPayments from './klarna-payments';
 import KlarnaWindow from './klarna-window';
+import KlarnaV2Window from './klarnav2-window';
 
-const SDK_URL = '//credit.klarnacdn.net/lib/v1/api.js';
```

Comment:
Here you removed sdk_url and you used in test

Reply:
actually I did not removed it I just change the constant name that's why I'm still doing unit test of it